src/s2_language_detection/language_classifier.py

```diff
@@ -5,13 +5,20 @@
 import fasttext
 import numpy.typing as npt
 
-FASTTEXT_FNAME = 'lid.176.bin'
+from s2_fos.constants import FASTTEXT_FNAME, PROJECT_ROOT_PATH
 
 
 class LanguageClassifier:
-    def __init__(self, data_dir):
+    def __init__(self, data_dir='data'):
```

Comment:
Same advice here. Default to `None`, otherwise use the exact user passed-in data_dir.